api/src/components/coauthor/service.ts

```diff
@@ -40,14 +40,26 @@ export const getAllByPublicationVersion = async (publicationVersionId: string) =
     return coAuthors;
 };
 
-export const update = (id: string, data: Prisma.CoAuthorsUpdateInput) =>
-    client.prisma.coAuthors.update({
+export const update = async (id: string, data: Prisma.CoAuthorsUpdateInput) => {
+    const updateCoAuthor = await client.prisma.coAuthors.update({
         where: {
             id
         },
         data
     });
 
+    await client.prisma.publicationVersion.update({
```

Comment:
We should use the update method exported from the publicationVersion/service.ts for all these updates

Reply:
Just pushed a commit that converted these calls to use the service